dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -57,12 +56,10 @@ Symbol getSymbol() {
   }
 
   /**
-   * Returns the referenced class that contains the symbol. Null when the target class is not found
-   * in the class path (this is the case if the errorType is {@code CLASS_NOT_FOUND} for top-level
-   * classes).
-   *
-   * <p>In case of a nested class is missing while its outer class is found in the class path, this
-   * method returns the outer class.
+   * Returns the class that references the symbol. If the symbol is a method or a field,
```

Comment:
```suggestion
   * Returns the class that contains the symbol. If the symbol is a method or a field,
```

Reply:
Not 100% sure, but I think "references" is clearer. "contains" sounds more to me like the class where the symbol is defined.

Reply:
This return value is the class file where the incompatible symbol is defined.

Reply:
is expected to contain
(since it doesn't contain the symbol, and that's the problem)

Reply:
Nice.